devolutions-gateway/src/api/monitoring.rs

```diff
@@ -0,0 +1,49 @@
+use axum::debug_handler;
+use axum::Json;
+use axum::Router;
+use axum::extract;
+use axum::routing;
+use axum::http;
+use network_monitor::MonitorResult;
+use crate::DgwState;
+use network_monitor;
+use network_monitor::SetConfigError;
+
+pub fn make_router<S>(state: DgwState) -> Router<S> {
+    let router = Router::new()
+        .route("/config", routing::post(handle_set_monitoring_config))
+        .route("/log/drain", routing::post(handle_drain_log));
+
+    router.with_state(state)
+}
+
+#[debug_handler]
+async fn handle_set_monitoring_config(
```

Comment:
suggestion: Please, add the openapi/utoipa documentation. You can check the `crate::api::session` module or another one, for reference.